internal/services/horusec_api/horus_api_test.go

```diff
@@ -136,34 +136,34 @@ func TestServiceSendAnalysis(t *testing.T) {
 
 			wantErr: false,
 		},
-		{
-			name: "should send analysis with no errors",
-			args: args{
-				entity: &analysis.Analysis{
-					ID:        uuid.New(),
-					CreatedAt: time.Now(),
-					Status:    enumHorusec.Running,
-				},
-				config: cliConfig.New(),
-				beforeFunc: func() (*httptest.Server, *os.File) {
-					dir := t.TempDir()
-					filename := "cert.pem"
-					file, err := os.Create(filepath.Join(dir, filename))
-					assert.NoError(t, err)
-					assert.NotNil(t, file)
-					defer file.Close()
+		// {
```

Comment:
if is going to be commented is better to remove it

Reply:
Maybe in the future we can rollback this! What think add a commentary ?

Reply:
I agre with @iancardosozup. If we will not use this code we should delete. If we need rollback just revert this commit.

Reply:
ok, done :)